test/grabURITest.coffee

```diff
@@ -0,0 +1,20 @@
+Helper = require('hubot-test-helper')
+helper = new Helper('../scripts/grabURI.coffee')
+
+expect = require('chai').expect
+co     = require('co')
+
+
+describe 'Otis', ->
+  context 'User posts URL', ->
+    room = helper.createRoom()
+    co =>
+      yield room.user.say('user1', 'http://fakewebsite.com')
```

Comment:
It's an artifact from all the problem solving of failure on the CI. I'll be removing it and refactoring and adding more tests.